composer.json

```diff
@@ -4,6 +4,7 @@
     "license": "MIT",
     "require": {
         "php": ">=7.1.0,<7.3.0",
+        "jetbrains/phpstorm-stubs": "2018.1.2",
```

Comment:
`2019.1` should be used now

Reply:
https://github.com/JetBrains/phpstorm-stubs/releases shows `2019.1` as the latest release whereas https://packagist.org/packages/jetbrains/phpstorm-stubs shows `2018.1.2`. Looks like their GitHub repo is missing the hook required for Packagist to update. CC @andrey-sokolov @neuro159

Reply:
@sebastianbergmann Sorry about the missing hook. Everything should be fine now.